src/main/java/com/ita/if103java/ims/dao/impl/UserDaoImpl.java

```diff
@@ -294,40 +284,40 @@ private PreparedStatement getPreparedStatement(User user, Connection connection)
                 SELECT *
                 FROM users
                 WHERE id = ?
+                AND active = 1
             """;
 
         public static final String SQL_SELECT_USER_BY_EMAIL = """
                 SELECT *
                 FROM users
                 WHERE email = ?
+                AND active = 1
             """;
 
         public static final String SQL_SELECT_ALL_USERS = """
                 SELECT *
                 FROM users
+                WHERE account_id = ?
+                AND active = 1
                 ORDER BY ?
                 Limit ?
                 Offset ?
             """;
 
-        public static final String SQL_SELECT_USERS_BY_ACCOUNT_ID = """
-                SELECT *
-                FROM users
-                WHERE account_id = ?
-            """;
-
         public static final String SQL_SELECT_WORKERS_BY_ACCOUNT_ID = """
                 SELECT *
                 FROM users
                 WHERE role = 'ROLE_WORKER'
                 AND account_id = ?
+                AND active = 1
             """;
 
         public static final String SQL_SELECT_ADMIN_BY_ACCOUNT_ID = """
                 SELECT *
                 FROM users
                 WHERE role = 'ROLE_ADMIN'
                 AND account_id = ?
+                AND active = 1
             """;
 
         public static final String SQL_UPDATE_USER = """
```

Comment:
```
public static final String SQL_SET_ACTIVE_STATUS_USER = """
                UPDATE users
                SET active = ?
                WHERE id = ?
            """;
```

Here also by account id and for hard delete as well

Reply:
I've secured these endpoints in another way

Reply:
Any troubles to update query?

Reply:
I've secured the delete method at the controller layer and here it is useless.

Reply:
What do you think about it?

Reply:
I saw, you approach. For this particular case it is not efficient.
For findById I agree, partially. But for soft delete not.

Reply:
Ok

Reply:
Thanks)